modules/sandboxed-containers-uninstalling-kata-runtime.adoc

```diff
@@ -0,0 +1,18 @@
+//Module included in the following assemblies:
+//
+// *disabling-sandboxed-container-workloads.adoc
+
+[id="sandboxed-containers-uninstalling-kata-runtime_{context}"]
+
+= Uninstalling Kata Runtime
+
+This section describes how to remove and uninstall the `kata` runtime and all
+its related resources (for example, CRI-O config, RuntimeClass, and so on) from
+your cluster.
+
+.Procedure
+
+1. Delete the `KataConfig` custom resource using `oc delete kataconfig <KataConfig_CR_Name>`.
+
+The OpenShift sandboxed containers operator removes all the resources initially created to enable the runtime on your cluster.
```

Comment:
s/resources initially/resources that were initially
(complementizer use best practice for L10N)